### PR TITLE
chore: upgrade template package version

### PR DESCRIPTION
**Pull request by karasusan**

### CODE REVIEW

**Comment by kannan-xiao4** [comment]
Optional: I think we used to symbolic link about documentation.

File: com.unity.renderstreaming/Documentation~/index.md [hunk 1]
```diff
@@ -8,7 +8,7 @@ title: index
 This package contains public APIs and project samples that are built on top of our [**Unity Render Streaming**](../../com.unity.template.renderstreaming-hd/Packages/com.unity.template.renderstreaming/Documentation~/index.md).
 
 > [!WARNING]
-> So this version is in development, please do not use it. The latest version is `1.2.2`.
+> So this version is in development, please do not use it. The latest version is `1.2.3`.
```
Comment (sindharta) [inline com.unity.renderstreaming/Documentation~/index.md:11]: ~So~ 
Because

File: com.unity.renderstreaming/README.md [hunk 2]
```diff
@@ -2,7 +2,7 @@
 
 <img src="https://img.shields.io/badge/unity-2019.3-green.svg?style=flat-square" alt="unity 2019.3">
 
-| :warning: | So this version is in development, please do not use it. The latest version is `1.2.2`. <br/> You can check the version you are using on the Package Manager. <br/> The release date of 2.0 is planned at the end of April 2020. |
+| :warning: | So this version is in development, please do not use it. The latest version is `1.2.3`. <br/> You can check the version you are using on the Package Manager. <br/> The release date of 2.0 is planned at the end of April 2020. |
```
Comment (sindharta) [inline com.unity.renderstreaming/README.md:5]: ~So~ 
Because

File: com.unity.renderstreaming/package.json [hunk 3]
```diff
@@ -3,7 +3,7 @@
   "displayName": "Unity Render Streaming",
   "version": "2.0.0-preview.1",
   "unity": "2019.3",
-  "description": "***\nWarning: So this version is in development, please do not use it. The latest version is 1.1.2.\n***\n\nThis is a package for using Unity Render Streaming technology. It contains two samples to use the technology.",
+  "description": "***\nWarning: So this version is in development, please do not use it. The latest version is 1.2.3.\n***\n\nThis is a package for using Unity Render Streaming technology. It contains two samples to use the technology.",
```
Comment (sindharta) [inline com.unity.renderstreaming/package.json:6]: ~So~ 
Because

**Pull request merged**